seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -2636,7 +2636,7 @@
         "<blockquote>function plusThree(num) {<br>  return num + 3;<br>}<br>var answer = plusThree(5); // 8</blockquote>",
         "<code>plusThree</code> takes an <dfn>argument</dfn> for <code>num</code> and returns a value equal to <code>num + 3</code>.",
         "<hr>",
-        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value."
+        "Create a function <code>timesFive</code> that accepts one argument, multiplies it by <code>5</code>, and returns the new value. You can call the <code>timesFive</code> function to test it."
```

Comment:
@erictleung Good point, I didn't read well enough 😅
The log should be visible in the output window, I believe that the message displaying is a bug.

How about making the seed look somewhat like this:
```js
// Example
function minusSeven(num) {
  return num - 7;
}

// Only change code below this line


var yourAnswer = timesFive(7); // change the number on this line to test your function
console.log(yourAnswer);

```
❓